README.md

```diff
@@ -187,36 +192,35 @@ For Arabic, a right-to-left language:
 
 We maintain a modern codebase that supports the latest two versions of evergreen web browsers. The current list of browsers officially supported by Spectrum CSS can be found in the browserslist section of the project's [package.json file](https://github.com/adobe/spectrum-css/blob/main/package.json). This setting is used by the build tools when the source files are built. If you require additional browser support for your project, the CSS can be processed further with your chosen tools.
 
-* latest 2 Edge versions
-* latest 2 Chrome versions
-* latest 2 Firefox versions
-* latest 2 Safari versions
-* latest 2 iOS versions
+- latest 2 Edge versions
+- latest 2 Chrome versions
+- latest 2 Firefox versions
+- latest 2 Safari versions
+- latest 2 iOS versions
 
 ### Optimizing Spectrum CSS
 
 Spectrum CSS is designed to be as modern and flexible as possible, and as such, leaves room for consumers to optimize in their own way. There are many tools and techniques you can use to optimize CSS for your project such as tree shaking, purging, and minification. If you are loading the entire set of `@spectrum-css/tokens` for example, you can safely tree shake the tokens to only include the variables you are using, often leading to a significant reduction in file size.
 
 ## Contributing
 
-A very special thank you to all of our [contributors](<https://github.com/adobe/spectrum-css/graphs/contributors>) without whom this project would not be possible.
+A very special thank you to all of our [contributors](https://github.com/adobe/spectrum-css/graphs/contributors) without whom this project would not be possible.
 
 Want to join the team? Check out the [contributing guidelines](.github/CONTRIBUTING.md) to get started.
 
 ## Tasks
 
 The following tasks are available:
 
-| Command | Description | Examples |
-| --- | --- | --- |
-| `clean` | Cleans all output files for the project and all components | `yarn clean` |
-| `build` | Performs a build of all components | `yarn build` |
-| `build:all` | Performs a build of all components, documentation site, and storybook | `yarn build:all` |
```

Comment:
Was this `yarn build:all` an old command that we just don't use anymore?

Reply:
Yeah I didn't even see it in the package.json anymore so it must have been removed earlier and just not cleaned up from the docs.